src/AppBundle/Repository/CommitteeMembershipRepository.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\CommitteeMembership;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\QueryBuilder;
+use Ramsey\Uuid\Uuid;
+
+class CommitteeMembershipRepository extends EntityRepository
+{
+    /**
+     * Returns whether or not the given adherent is already an host of at least
+     * one committee.
+     *
+     * @param string $adherentUuid
+     *
+     * @return bool
+     */
+    public function isCommitteeHost(string $adherentUuid): bool
+    {
+        $adherentUuid = Uuid::fromString($adherentUuid);
+
+        $query = $this
+            ->createQueryBuilder('cm')
+            ->select('COUNT(cm.uuid)')
+            ->where('cm.adherentUuid = :adherent')
+            ->andWhere('cm.privilege = :privilege')
+            ->setParameter('adherent', (string) $adherentUuid)
+            ->setParameter('privilege', CommitteeMembership::COMMITTEE_HOST)
+            ->getQuery()
+        ;
+
+        return (int) $query->getSingleScalarResult() >= 1;
```

Comment:
Yes you're right this is because PHP is shit with numbers vs bool vs strings. IMO, from a cognitive perspective, it requires less efforts to compare numbers together rather than casting to a boolean. Also it's safer!